docs/csharp/language-reference/compiler-messages/cs8892.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Compiler warning (level 5) CS8892"
+ms.date: 07/20/2015
+f1_keywords: 
+  - "CS8892"
+helpviewer_keywords: 
+  - "CS8892"
+author: Youssef1313
+---
```

Comment:
I see other compile-messages have ms.assetid in the metadata. Is it required? What is it for? How to get the correct value for it?

Reply:
That's a historical artifact from the migration from MSDN. It's no longer needed.